packages/web-components/fast-foundation/src/disclosure/disclosure.template.ts

```diff
@@ -0,0 +1,24 @@
+import { html, ref } from "@microsoft/fast-element";
+import { Disclosure } from "./disclosure";
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#Disclosure} component.
+ * @public
+ */
+export const DisclosureTemplate = html<Disclosure>`
+    <template>
```

Comment:
Do we need this template element if we aren't using it?

Reply:
Not anymore.